modules/planning/tasks/deciders/lane_change_decider/lane_change_decider.cc

```diff
@@ -188,37 +193,53 @@ void LaneChangeDecider::UpdateStatus(double timestamp,
 }
 
 void LaneChangeDecider::PrioritizeChangeLane(
+    const bool is_prioritize_change_lane,
     std::list<ReferenceLineInfo>* reference_line_info) const {
   if (reference_line_info->empty()) {
     AERROR << "Reference line info empty";
     return;
   }
+  // TODO(SHU): disable the reference line order change for now
+  return;
```

Comment:
Add a FLAG_enable_reference_line_order which default to false, and
```
if (!FLAG_enable_reference_line_order) {
  return;
}
```
So this piece of code seems better.